keystone_api/plugins/slurm.py

```diff
@@ -47,55 +47,56 @@ def get_accounts_on_cluster(cluster_name: str) -> Collection[str]:
     return out.split()
 
 
-def set_cluster_limit(account_name: str, cluster_name: str, limit: int, in_minutes: bool = False) -> None:
-    """Update the current TRES Billing hour usage limit to the provided limit on a given cluster for a given account
-    with sacctmgr. The default time unit is Hours.
+def set_cluster_limit(account_name: str, cluster_name: str, limit: int, in_hours: bool = True) -> None:
+    """Update the current TRES Billing usage limit to the provided limit on a given cluster for a given account
+    with sacctmgr. The default expected limit unit is Hours, and a conversion takes place as Slurm uses minutes.
 
     Args:
         account_name: The name of the account to get usage for
         cluster_name: The name of the cluster to get usage on
         limit: Number of billing TRES hours to set the usage limit to
-        in_minutes: Boolean value for whether (True) or not (False) the set limit is in minutes (Default: False)
+        in_hours: Boolean value for whether (True) or not (False) the limit provided is in Hours (Default: True)
     """
 
-    if in_minutes:
+    # Convert the input hours to minutes
+    if in_hours:
         limit *= 60
 
-    cmd = split(f"sacctmgr modify account where account={account_name} cluster={cluster_name} set GrpTresRunMins=billing={limit}")
+    cmd = split(f"sacctmgr modify -i account where account={account_name} cluster={cluster_name} set GrpTresMins=billing={limit}")
```

Comment:
`-i` on these commands means there is no prompt for the user to confirm. 

GrpTresMins has the desired behavior of acting as a limit on usage across all jobs since the last rawusage reset. GrpTresRunMins only acts on running jobs, and has only worked this whole time because we've only ever been setting it between 0 and -1.